src/frontends/pytorch/src/frontend.cpp

```diff
@@ -0,0 +1,143 @@
+            auto op_type = fw_node->get_decoder()->get_op_type();
+            unconverted_ops_types.insert(op_type);
+        }
+        if (const auto& fw_node = ov::as_type_ptr<ov::op::util::MultiSubGraphOp>(node)) {
+            for (int i = 0; i < fw_node->get_internal_subgraphs_size(); i++) {
+                auto internal_types = get_unconverted_types_from_model(fw_node->get_function(i));
+                unconverted_ops_types.insert(internal_types.begin(), internal_types.end());
+            }
+        }
+    }
+    return unconverted_ops_types;
+}
+}  // namespace
+
+std::shared_ptr<Model> FrontEnd::convert(const InputModel::Ptr& model) const {
+    auto converted_model = convert_partially(model);
+    normalize(converted_model);
+    std::set<std::string> unconverted_ops_types = get_unconverted_types_from_model(converted_model);
+    std::stringstream ops_str;
+    for (auto&& op_type : unconverted_ops_types) {
+        ops_str << op_type << "\n";
+    }
+    FRONT_END_OP_CONVERSION_CHECK(unconverted_ops_types.size() == 0,
+                                  "Model wasn't fully converted. Unconverted operation types:\n" + ops_str.str());
+    return converted_model;
+}
+
+void FrontEnd::convert(const std::shared_ptr<Model>& partiallyConverted) const {
+    FRONT_END_NOT_IMPLEMENTED(convert);
+}
+
+std::shared_ptr<Model> FrontEnd::convert_partially(const ov::frontend::InputModel::Ptr& model) const {
+    try {
+        auto pytorch_model = std::dynamic_pointer_cast<pytorch::InputModel>(model);
+        auto model = convert_pytorch_model(pytorch_model->m_model);
+
+        return model;
+    } catch (const std::runtime_error& e) {
+        std::cerr << "[ ERROR ] Unexpected error while converting pytorch model: " << e.what() << "\n";
+        std::cerr << "Rethrowing. Misleading error message from pybind11 may come next. TODO.";
+        throw;
+    }
+}
+
+std::shared_ptr<Model> FrontEnd::decode(const InputModel::Ptr& model) const {
+    FRONT_END_NOT_IMPLEMENTED(decode);
+}
+
+void FrontEnd::normalize(const std::shared_ptr<ov::Model>& model) const {
+    ov::pass::Manager manager;
+
+    manager.register_pass<ov::pass::ConstantFolding>();
+    manager.register_pass<ov::pass::UnrollIf>();
+    // Have to run UnrollIf second time, because conditions are defined outside of nested If (ticket 98155)
+    manager.register_pass<ov::pass::UnrollIf>();
+    manager.register_pass<ov::frontend::pytorch::pass::AtenCatToConcat>();
+    manager.register_pass<ov::frontend::pytorch::pass::AppendListUnpackReplacer>();
+    manager.register_pass<ov::frontend::pytorch::pass::PrimListUnpackReplacer>();
+    manager.register_pass<ov::frontend::pytorch::pass::AtenGetItemReplacer>();
+    manager.register_pass<ov::frontend::pytorch::pass::MaxPrimListConstructReplacer>();
+    manager.register_pass<ov::frontend::pytorch::pass::DecomposeTupleResults>();
+    manager.register_pass<ov::pass::ConstantFolding>();
```

Comment:
why do we need `ConstantFolding`? I think it is better to avoid and use `get_const_from_source` function by demand. Otherwise, it increases conversion time.

Reply:
Looks reasonable. Honestly, I don't remember why do we need it. Will address in the next PRs if it is not necessary here.